docs/core/diagnostics/built-in-metrics-aspnetcore.md

```diff
@@ -67,6 +67,8 @@ The time ends when:
 - All response data has been sent.
 - The context data structures for the request are being disposed.
 
+The default buckets for this metic are set to { 0, 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 }.
```

Comment:
```suggestion
When using OpenTelemetry, the default buckets for this metric are set to { 0, 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 }.
```
In the future hopefully buckets can be associated directly with the metric, but for now anyone not using OpenTelemetry isn't going to see them. Same thing for all the other places this text shows up.